programs/server/users.xml

```diff
@@ -5,6 +5,8 @@
     <profiles>
         <!-- Default settings. -->
         <default>
+            <allow_introspection_functions>1</allow_introspection_functions>
```

Comment:
This does not look required for the test. Anyway same comment as above that this is not the correct place to modify settings for tests.